accounts-db/src/accounts_db.rs

```diff
@@ -2399,13 +2399,13 @@ impl<'a> AppendVecScan for ScanState<'a> {
             } else if self.config.check_hash && computed_hash != loaded_hash {
                 info!(
                     "hash mismatch found: computed: {}, loaded: {}, pubkey: {}",
-                    computed_hash, loaded_hash, pubkey
+                    computed_hash.0, loaded_hash.0, pubkey
                 );
                 self.mismatch_found.fetch_add(1, Ordering::Relaxed);
             }
         }
         let source_item = CalculateHashIntermediate {
-            hash: loaded_hash,
+            hash: loaded_hash.0,
```

Comment:
is this preferred over `*&loaded_hash`? I'm not sure.

Reply:
maybe `.into()`? I'm not sure the cleanest. I've never liked `.0`. I find it hard to remember what .0 is. Not a deal breaker. Just brainstorming on the best api.